.github/workflows/chromatic.yml

```diff
@@ -87,6 +88,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 2 # we need to fetch at least parent commit to satisfy Chromatic
+          ref: ${{ github.event.pull_request.head.sha || github.sha }} # HEAD commit instead of merge commit
```

Comment:
I was using this kind of override in the past but lately, I reverted that decision in my projects. While the default behavior of using the merge commit is surprising at first, it actually might catch more pre-merge problems. YMMV, ofc.

Reply:
we are using lostpixel for visual regression tests, chromatic is only storybook, no tests, but the same question is probably for lostpixel config